documentation/modules/operators/proc-configuring-fips-mode-cluster-operator.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-cluster-operator.adoc
+
+[id='proc-configuring-fips-mode-cluster-operator-{context}']
+= Configuring the FIPS mode in Cluster Operator
+
+[role="_abstract"]
+When running Strimzi on FIPS enabled Kubernetes cluster, the OpenJDK used in Strimzi container images will automatically switch to FIPS mode.
+Strimzi currently doesn't support running as FIPS compliant with the FIPS mode enabled.
+When you deploy Strimzi on such cluster, you will see errors similar to this:
+
+[source%nowrap]
+----
+Exception in thread "main" io.fabric8.kubernetes.client.KubernetesClientException: An error has occurred.
+	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:103)
+	at io.fabric8.kubernetes.client.KubernetesClientException.launderThrowable(KubernetesClientException.java:97)
+	at io.fabric8.kubernetes.client.utils.HttpClientUtils.applyCommonConfiguration(HttpClientUtils.java:210)
```

Comment:
do we need to show all these lines?

Reply:
I can try to comment out some of the lines. But the exact exception might change between releases. So I do not think we can leave just the first line. And I'm not sure if it matters that we replace half of the lines with comments. But I can try that